docs/README.md

```diff
@@ -92,106 +107,176 @@ Now, you are able to use all blocks exported by the `recommendation-shelf` app.
-| `strategy`           | `enum`    | Strategy that will be used to fetch and display the recommended products. Only possible value is `BOUGHT_TOGETHER` (returns products bought together with the first search products or the PDP product.)       | `BOUGHT_TOGETHER` |
 | `recommendation`     | `object`  | Settings for the recommendation shelf.  | `undefined` |
-
-- `recommendation` object:
-
+ 
+### `recommendation` object
+ 
 | Prop name            | Type      | Description                                                                      | Default value |
 | -------------------- | --------- | -------------------------------------------------------------------------------- | ----- |
-| `count`              | `object`  | Defines the total and minimum number of recommendations that should be fetched. | `{minimum: 5, recommendations: 20}` |
-
-- `count` object:
-
+| `count`              | `object`  | Defines the total and the minimum number of recommendations that should be fetched. | `{minimum: 5, recommendations: 20}` |
+ 
+### `count` object
+ 
 | Prop name         | Type      | Description                                                    | Default value |
 | ----------------- | --------- | -------------------------------------------------------------- | ------------- |
 | `minimum`         | `number`  | Defines the minimum recommendations that should be fetched.   | `5`             |
 | `recommendations` | `number`  | Defines the total number of recommendations that should be fetched. | `20`            |
-
+ 
+## Setting up the `recommendation-refresh`
+ 
+To configure the `recommendation-refresh` you will need to add props to it. Check the following tables to know what props that the app allows and how to configure them.
+ 
 ### `recommendation-refresh` props
-
+ 
 | Prop name            | Type      | Description                                                                      | Default value      |
 | -------------------- | --------- | -------------------------------------------------------------------------------- | ------------------ |
-| `strategy`           | `enum`    | Strategy that will be used to fetch and display the recommended products. Possible values can be find in the table below.   | `RECOMMENDATION_HISTORY` |
-| `secondaryStrategy`  | `enum`    | Secondary strategy that will be used to fetch and display the recommended products if the initial strategy does not return results. Possible values can be find in the table below. | `RECOMMENDATION_HISTORY` |
+| `strategy`           | `enum`    | Strategy used to fetch and display the recommended products. Possible values can be found in the [table](link pra tabela de estratégias).   | `RECOMMENDATION_HISTORY` |
+| `secondaryStrategy`  | `enum`    | Secondary strategy used to fetch and display the recommended products if the initial strategy does not return results. Possible values can be found in the [table](link pra tabela de estratégias). | `RECOMMENDATION_HISTORY` |
```

Comment:
same. remembering that the possible strategies for the `recommendation-refresh` are not the same as the `recommendation-shelf`, so we need to create the table with the possible strategies for it.

Reply:
it's easier to say the strategies that apply to it hahaha 
the only ones that work with this shelf are: `RECOMMENDATION_HISTORY`,` ORDER_HISTORY` and `CART_HISTORY`

Reply:
Done! ✔️